src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,18 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int VALID_AGE = 35;
+    private static final String VALID_NATIONALITY = "Ukrainian";
+    private static final int IN_UKR = 10;
```

Comment:
не використовуй скорочень 
```suggestion
    private static final int IN_UKRAINE = 10;
```

Reply:
Тогда строка не влезет в 100

Reply:
ти не змінив в методі на IN_UKRAINE